src/set_target_node.cpp

```diff
@@ -0,0 +1,112 @@
+#include <rt_assignment_2/set_target_node.hpp>
+#include <rt_assignment_2/RobotState.h>
+#include <rt_assignment_2/RobotTarget.h>
+
+#include <ros/ros.h>
+#include <ros/timer.h>
+
+#include <boost/bind.hpp>
```

Comment:
You are using C++11, `std::bind` and `std::placeholder` are in standard since then and can be used with ROS. You can access them by importing `<functional>`